.travis.yml

```diff
@@ -77,37 +77,40 @@ jobs:
       after_success:
       - codecov
 
-    - stage: e2e
+    - stage: "Test"
       name: "containerd - calico e2e Tests"
       language: python
       cache: pip
       python: "3.8"
       env:
       - KUBECONFIG="$HOME/.kube/config"
       - CONTAINERD_VERSION=1.3.4
-      # TODO upgrade to v1.18.6 or newer
-      - KUBERNETES_VERSION=v1.15.3
-      # TODO update to v1.12.1 or newer
-      - MINIKUBE_VERSION=v1.3.1
+      # stable version in minikube v1.12.1
+      - KUBERNETES_VERSION=v1.18.3
+      - MINIKUBE_VERSION=v1.12.1
+      - DOCKER_REGISTRY=localhost:5000
       - CHANGE_MINIKUBE_NONE_USER=true
       - MINIKUBE_WANTUPDATENOTIFICATION=false
       - MINIKUBE_WANTREPORTERRORPROMPT=false
       - MINIKUBE_HOME=$HOME
+      - CONTAINER_RUNTIME_ENDPOINT=unix:///run/containerd/containerd.sock
+      - IMAGE_SERVICE_ENDPOINT=unix:///run/containerd/containerd.sock
```

Comment:
Actually I don't know if these are the right sockets since docker is already installed and running on the machine (this would be solved by the `docker` driver).